tests/e2e/utils/flows.ts

```diff
@@ -2,20 +2,19 @@
  * @format
```

Comment:
I suppose we could eventually do away with this as well given it's mostly in page objects now, aside from the extra login/logout logic.

Reply:
@louwie17 Yeah its on my hit list. 🔫 I'll probably remove this as part of this PR

Reply:
I've removed this altogether.